docs/content/latest/yugabyte-platform/security/create-kms-config/equinix-smartkey.md

```diff
@@ -28,6 +28,13 @@ showAsideToc: true
     </a>
   </li>
 
+  <li >
+    <a href="{{< relref "./hashicorp-kms.md" >}}" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      HashiCorp Vault
+    </a>
+  </li>
+
 </ul>
 
 Encryption at rest uses universe keys to encrypt and decrypt universe data keys. You can use the Yugabyte Platform console to create key management service (KMS) configurations for generating the required universe keys for one or more YugabyteDB universes. Encryption at rest in Yugabyte Platform supports the use of [Equinix SmartKey](https://www.equinix.com/services/edge-services/smartkey/) as a KMS.
```

Comment:
Link appears to have lapsed, now redirecting to Fortanix site. But https://www.equinix.com/services/digital-infrastructure-services/smartkey seems to work

Reply:
Fixed